e2etest/pkg/routes/interfaces_test.go

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier:Apache-2.0
```

Comment:
not necessarily. I tested findInterfaceWithAddress because that was involving parsing and a logic which was quite complex, mostly to speed up the development and because it does not depend on the external `ip` command. And because this functions belong to e2e, so the part is already instrumented